tarofreighter/chain_porter.go

```diff
@@ -81,6 +93,8 @@ type ChainPorter struct {
 
 	exportReqs chan *AssetParcel
 
+	notifications
```

Comment:
If this struct is embedded anyway, I don't really see any reason for it. Even makes it slightly harder to read as compared to declaring `subscribers` and `subscriberMtx` here directly.
Also, if we declare it here directly, we can actually initialize the `subscribers` map in `NewChainPorter()` directly.

Reply:
I don't mind either way. Went for embedded following this comment: https://github.com/lightninglabs/taro/pull/235#discussion_r1080714456

I will switch back to directly adding the fields so that I can initialise the `subscribers` map in `NewChainPorter()`.